tools/find_untranslated_strings.py

```diff
@@ -0,0 +1,159 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
I had some unicode chars while working on the script, it wouldn't run without this declaration. Doesn't hurt to leave it in.